.helm-staging.yml

```diff
@@ -49,11 +49,9 @@ bblfshd-sidecar:
     tag: v2.12.0
   drivers:
     languages:
-      # for js we have to lock old version due to style-analyzer
       javascript:
         repository: bblfsh/javascript-driver
-        tag: v1.2.0
-      # for the rest we can use latest
+        tag: v2.7.1
```

Comment:
`2.7.2` is already available.
https://hub.docker.com/r/bblfsh/javascript-driver/tags
We could maybe update to the current latest one?

Reply:
Yes, 2.7.2 is the one included in the bblfsh/bblfshd:v2.12.0-drivers image, we should be using than one please. Unless 2.7.1 is a hard requirement from @src-d/machine-learning.

Reply:
I think it should be good to use 2.7.2, nothing serious was changed there.